partials/tabs/tryonline.html

```diff
@@ -1,43 +1,205 @@
-<div>
+<div class="coala-online">
   <div class="row">
     <div class="col m8 offset-m2 s12">
       <form class="coa-form">
-        <textarea placeholder="Write some code here" class="file-data bb-data pre small" name="file_data">print( a + b )
+        <textarea placeholder="Write some code here" id="file-data" class="file-data bb-data pre small" name="file_data">print( a + b )
 </textarea>  <!-- DONT EVER INDENT THIS, it ends up in the textarea -->
-        <div class="col s10 m11 l11">
-          <div class="chips chips-bears bb-chips" ng-show="toc.done_loading_bears"></div>
+      </form>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col m8 offset-m2 s12">
+      <select id="lang" ng-init="sc.lang_sel='Python'" ng-model="toc.lang_sel" ng-options="lang for lang in toc.LANGUAGES">
+        <option value="" disabled>Choose a Language</option>
+      </select>
+    </div>
+    <div class="m2" ng-show="loading_coajson">
+      <div class="preloader-wrapper small active">
+        <div class="spinner-layer spinner-blue">
+          <div class="circle-clipper left">
+            <div class="circle"></div>
+          </div><div class="gap-patch">
+            <div class="circle"></div>
+          </div><div class="circle-clipper right">
+            <div class="circle"></div>
+          </div>
         </div>
-        <div class="col s2 m1 l1">
-          <button class="btn-floating waves-effect" type="submit" ng-click="toc.submit_coa_form()"><i class="material-icons right">send</i></button></div>
+        <div class="spinner-layer spinner-red">
+          <div class="circle-clipper left">
+            <div class="circle"></div>
+          </div><div class="gap-patch">
+            <div class="circle"></div>
+          </div><div class="circle-clipper right">
```

Comment:
General Formatting: Use a new line for every block, list, or table element, and indent every such child element: Move the opening "div" to its own line.

*HTMLLintBear, severity MAJOR, section `html`.*